src/components/NavBar.js

```diff
@@ -27,26 +27,102 @@ const Icon = styled.i`
   font-size: 2rem;
 `;
 
-const NavBar = () => {
-  return (
-    <NavBarContainer>
-      <StyledLink to="/LikedCareers">
-        <NavBarItem>
-          <Icon className="material-icons">playlist_add_check</Icon>
-        </NavBarItem>
-      </StyledLink>
-      <StyledLink to="/Careers">
-        <NavBarItem>
-          <Icon className="material-icons">view_carousel</Icon>
-        </NavBarItem>
-      </StyledLink>
-      <StyledLink to="/Settings">
-        <NavBarItem>
-          <Icon className="material-icons">settings</Icon>
-        </NavBarItem>
-      </StyledLink>
-    </NavBarContainer>
-  );
-};
+const SettingsContainer = styled.div`
+  color: black;
+  background-color: white;
+  width: 100%;
+  display: ${props => (props.settings ? 'block' : 'none')};
+  position: absolute;
+  top: 4rem;
+  right: 0;
+  box-shadow: 0 0.7rem 1.2rem rgba(0, 0, 0, 0.09),
+    0 0.4rem 0.4rem rgba(0, 0, 0, 0.16);
+`;
+
+const Wrapper = styled.div`
+  position: relative;
+`;
+
+const SettingsList = styled.ul``;
+
+const SettlingsListItem = styled.li`
+  height: 4rem;
+  border-bottom: 0.1rem solid black;
+  border-width: thin;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  font-size: 1.5rem;
```

Comment:
I think this should be more like `1.2rem` and also maybe tone the `color` down to a dark grey instead of black.